### PR TITLE
sidekiq background jobs

### PR DESCRIPTION
**Pull request by HASNAT204**

### CODE REVIEW

File: spec/mailers/previews/seller_mailer_preview.rb
Comment (sajidali-dnt) [inline spec/mailers/previews/seller_mailer_preview.rb:1]: why is this file present? are you going to write test cases?

**Review by sajidali-dnt — CHANGES_REQUESTED** [review]

File: app/jobs/send_seller_email_job.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class SendSellerEmailJob < ApplicationJob
+  queue_as :default
+
+  def perform(order_id)
+    order = Order.find(order_id)
+    order.order_items.each do |order_item|
```
[the remark this answers is not in the document]
Comment (HASNAT204) [reply app/jobs/send_seller_email_job.rb]: ok

File: spec/mailers/previews/seller_mailer_preview.rb
Comment (HASNAT204) [reply spec/mailers/previews/seller_mailer_preview.rb:1]: it was created by default when generating mailer and now i have removed it.

File: app/controllers/orders_controller.rb [hunk 2]
```diff
@@ -49,7 +50,7 @@ def create
   private
 
   def check_buyer_role
-    return if current_user.has_role?('Buyer')
+    return if current_user.has_role("Buyer")
```
[the remark this answers is not in the document]
Comment (HASNAT204) [reply app/controllers/orders_controller.rb]: yes this issue is now fixed.

**Review by sajidali-dnt — APPROVED** [review]

**Pull request merged**